ocaml/typing/typetexp.ml

```diff
@@ -844,9 +838,37 @@ and transl_type_aux env policy mode styp =
   | Ptyp_extension ext ->
       raise (Error_forward (Builtin_attributes.error_of_extension ext))
 
-and transl_type_aux_jst _env _policy _mode _attrs
+and transl_type_aux_jst env policy mode attrs loc ctyp
```

Comment:
It is a bug to take `ctyp` as an argument here, because it captures the attributes on the original parsetree type, but we want to strip the jane syntax attributes for the typed tree version.